## Section 3.1, 3.2

In [1]:
import pandas as pd
import numpy as np

data = [15,14,13,16,17,14,14,15,12,6,20,14,13,14]

In [2]:
df = pd.DataFrame(data)

In [3]:
df.describe()

,0
count,14.000000
mean,14.071429
std,3.049950
min,6.000000
25%,13.250000
50%,14.000000
75%,15.000000
max,20.000000


You can get the individual items from the DataFrame object.

In [4]:
df.mean()

0    14.071429
dtype: float64

In [5]:
df.var()

0    9.302198
dtype: float64

In [6]:
df.std()

0    3.04995
dtype: float64

We discussed modes in class

In [7]:
df.mode()

,0
0,14


The coefficient of variance would be given by:

In [10]:
cv = (df.std() / df.mean()) * 100
cv

0    21.674775
dtype: float64

So the coefficient of variance is 21.67%

There is also a sub-module of numpy that handles statistics. Also, since Python 3.4, there is a statistics module as part of the standard library.